src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ExceptionRecord.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class ExceptionRecord {
+
+    @JsonProperty("journeyClassification")
+    public final String classification;
+
+    @JsonProperty("poBox")
+    public final String poBox;
+
+    @JsonProperty("poBoxJurisdiction")
+    public final String jurisdiction;
+
+    @JsonProperty("deliveryDate")
+    public final LocalDateTime deliveryDate;
+
+    @JsonProperty("openingDate")
+    public final LocalDateTime openingDate;
+
+    @JsonProperty("scannedRecords")
```

Comment:
we should settle on a field name?
records or documents and be consistent in the 2 spreadsheets?